### PR TITLE
Move six from tests/requirements.txt to requirements.txt

### PR DESCRIPTION
**Pull request by toabctl**
six is used in plugins/actions/config_template.py and
roles/ceph-common/plugins/actions/config_template.py so move it to
requirements.txt .

### CODE REVIEW

**Comment by guits** [comment]
jenkins test centos-container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-purge

**Comment by guits** [comment]
jenkins test centos-non_container-switch_to_containers

**Comment by guits** [comment]
@toabctl please, add a sign off

**Comment by guits** [comment]
@toabctl could you rebase on master please?

**Comment by toabctl** [comment]
> @toabctl please, add a sign off

@guits done

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # These are Python requirements needed to run ceph-ansible master
 ansible>=2.8,<2.9
 netaddr
+six==1.10.0
```
Comment (dsavineau) [inline requirements.txt]: I think you can drop the version here

Comment (toabctl) [reply requirements.txt]: Done

**Comment by toabctl** [comment]
> please, squash your commits

@guits done

File: ceph-ansible.spec.in [hunk 2]
```diff
@@ -22,9 +22,11 @@ Requires: ansible >= 2.8
 %if 0%{?rhel} == 7
 BuildRequires: python2-devel
 Requires: python2-netaddr
+Requires: python3-six
```
Comment (dsavineau) [inline ceph-ansible.spec.in]: python2 here

**Comment by guits** [comment]
jenkins test centos-container-podman

**Comment by guits** [comment]
jenkins test centos-container-podman

**Pull request closed without merging**